### ISSUE TITLE
[FEATURE] Method to set admin user

### ISSUE THREAD

**Issue by WirtsLegs** [jordan-dalby/ByteStash#125]
**Is your feature request related to a problem? Please describe.**
Currently admin is user with id 1, this si a problem if you transitioned to SSO after initially having a test user, or if you for any other reason need to change the admin user

**Describe the solution you'd like**
Add a method to change the admin user, ideas would be
1) a script in the container we can exec with a username to change the admin
2) a env var we can set with the admin's name
and then something that would be good regardless of other options used
3) Be able to set a group name that if the user is a member of (from idP/OIDC claims) they have admin access.

**Describe alternatives you've considered**
Only fix right now is to destroy the instance and rebuild being careful to make the first user the intended admin, not ideal for obvious reasons

**Additional context**
No additional context

**Comment by jordan-dalby** [comment]
This should go hand-in-hand with an admin panel of some description, I'm thinking:
- Manage users, make people standard users / admin
- That's it, don't ask me why this is a list

**Comment by WirtsLegs** [comment]
> This should go hand-in-hand with an admin panel of some description, I'm thinking:
> 
>     * Manage users, make people standard users / admin
> 
>     * That's it, don't ask me why this is a list

yeah some way to manage other users would be good

just please if possible include some way to set a admin via a script or exec call in the container or w/e else so that logging in as the actual admin isn't required.

**Comment by WirtsLegs** [comment]
Another thing to consider on this, a way to change ownership of snippets

I was fiddling with integrating a social login to my IDP and deleted and re-created my account to test (giving my keycloak account a new id), now my snippets are not owned by me anymore and I have no way of fixing it.

**Comment by jordan-dalby** [comment]
@WirtsLegs Yeah perhaps a transfer all snippets to new account option, something like that.

**Comment by WirtsLegs** [comment]
> @WirtsLegs Yeah perhaps a transfer all snippets to new account option, something like that.

Probably just like an admin option to delete an account, when you do that you get a prompt on whether you want to delete the users snippets as well or transfer them to another account, if you select transfer you have to pick who to transfer to

**Comment by ados8** [comment]
A way to manage users definitely needed.
Setup OIDC and there's no way to manage e.g. delete users.
